### DISCUSSION TITLE
Energy dashboard Solis plugin -

### DISCUSSION THREAD

**Discussion by dewafre** [wills106/homeassistant-solax-modbus#211] [Q&A]
Hello,

I have a Solis inverter RHI-3P8K-HVES-5G and using the Solis plugin.
All the data is imported in 108 entities and now im trying to configure the Energy Dashboard as in the Wiki.
But when looking at the dashboard, some data looks very high. I think the **Solis Grid Export Today** and **Solis Grid Import Today** are scaled x10 comparing the online SolisCloud platform.

Anyone having the same issue or know what to do?
Thx alot for the help.

![image](https://user-images.githubusercontent.com/64728205/202904346-83113426-fc6d-436d-b726-a71b93d9f97d.png)

![image](https://user-images.githubusercontent.com/64728205/202904448-6a3c3dcb-aae6-423e-8c16-b0ba436ac82c.png)

![image](https://user-images.githubusercontent.com/64728205/202905260-4d369871-3e64-48ed-b337-e925c5ea917e.png)

**Energy from and to the net**
![image](https://user-images.githubusercontent.com/64728205/202904724-44f636b5-fb20-42f1-85b5-d7b8a8af1667.png)

**Comment by solarjones** [comment]
My readings are very similar (abnormal). I'm just in the process of deinstalling and reinstalling the SolaX integration to see if the latest version fixes this.

**Reply by wills106** [reply]
I haven't updated to Integration yet, so it will be multiplied by 10 for the time being.

https://github.com/wills106/homeassistant-solax-modbus/issues/210
